### ISSUE TITLE
[Bug] Pasting from google keyboard section doesn't make it convert into preview block

### ISSUE THREAD

**Issue by richardshiue** [AppFlowy-IO/AppFlowy#6477]
### Bug Description

Title says all

### How to Reproduce

1. Copy a URL
2. go back to appflowy, click on a random spot to begin editing
3. press on the paste button from the keyboard

### Expected Behavior

The link should be automatically converted into a preview block

### Operating System

Android 14

### AppFlowy Version(s)

0.7.1

### Screenshots

https://github.com/user-attachments/assets/b4a19e19-514a-4167-80ae-f8ab576a5322

**Comment by Donatzsky** [comment]
I don't think it matters how you paste the URL, it's simply not recognised. At least that's how it is for me.
